components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/org/wso2/carbon/idp/mgt/ui/i18n/Resources.properties

```diff
@@ -432,9 +432,11 @@ title.add.default.seq.select.mode=Select Mode
 script.default.seq.placeholder=Please add a default authentication sequence configuration in XML
 saml.metadata.validity.period=Metadata Validity Period
 saml.metadata.validity.period.help=Set the SAML metadata validity period in minutes
-saml.metadata.signing.enabled=Enable metadata signing
+saml.metadata.signing.enabled=Enable metadata Signing
```

Comment:
let's make M upper case in the resource value. it should be 'Enable Metadata Signing'